### DISCUSSION TITLE
UnicodeEncodeError: 'ascii' codec can't encode characters

### DISCUSSION THREAD

**Discussion by stabud** [argosopentech/argos-translate#163] [Q&A]
Hi!

I tried using terminal on linux with argos-translate:

`argos-translate.cli --from-lang en --to-lang ru "Hello World"`

And got the error:

> Traceback (most recent call last):
  File "/snap/argos-translate/10/bin/argos-translate", line 5, in <module>
    cli.main()
  File "/snap/argos-translate/10/lib/python3.6/site-packages/argostranslate/cli.py", line 60, in main
    print(translation.translate(text_to_translate))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)

When using the GUI, everything works well. However, I would like to be able to set the preferred default languages ​​when starting the GUI. At the moment by default: english - arabic.

**Comment by PJ-Finlay** [comment]
I reproduced this issue, it seems to be Snapcraft specific to trying to encode and print special characters (translating to Spanish works). I made an issue ticket for this and default languages.

- https://github.com/argosopentech/argos-translate/issues/164
- https://github.com/argosopentech/argos-translate/issues/165

**Reply by PJ-Finlay** [reply]
An immediate solution would be to install from PyPI or source.

**Reply by stabud** [reply]
Thank you!
